docx/oxml/numbering.py

```diff
@@ -138,37 +138,54 @@ def get_abstractNum(self, numId):
             if el.abstractNumId == abstractNum_id:
                 return el
 
+    def get_numId_lvl_for_p(self, p, styles_cache):
+        """
+        Returns tuple of `(numId, lvl)` where `numId` represent identifier which
+        references to numbering instance, and `lvl` object represents relevant level of
+        numbering scheme necessary information about paragraph indentation level and formating.
```

Comment:
I think this should be reworded a little bit. It is hard to comprehend on a first reading.